administrator-manual/en/release_notes.rst

```diff
@@ -24,12 +24,28 @@ Release notes |version|
 
 .. only:: nsent
 
-    - ISO release 7.8.2003 "final" replaces any previous ISO 7.7.1908
+    - ISO release 7.9.2009 "final" replaces any previous ISO
 
-    - This release is based on `CentOS 7 (1908) <https://wiki.centos.org/Manuals/ReleaseNotes/CentOS7>`_
+    - This release is based on `CentOS 7 (2009) <https://wiki.centos.org/Manuals/ReleaseNotes/CentOS7>`_
 
     - CentOS 7 will receive security updates until 2024-06-30
 
+Major changes on 2020-11-XX
+---------------------------
+
+* ISO release 7.9.2009 "final" replaces any previous ISO 7.8.2003
+
+* New installations of Nextcloud honor the StartTLS setting of the Active Directory accounts provider.
+  As old installations ignore that setting and always send clear-text passwords, it is recommended
+  to upgrade an existing installation to the new behavior, if its remote AD accounts provider
```

Comment:
```suggestion
  to upgrade an existing installation to the new behavior. If its remote AD account provider
```

Reply:
I was thinking the same...

What do you think to remove completely "If remote AD account provider supports StartTLS"? It should be true if sssd is configured for it.

Reply:
I do not have too much faith on Microsoft AD.
What about rephrasing to:
```
Make sure the remote AD supports StartTLS, then run the following command ::
```

Then add also a line like "NethServer AD always support StartTLS".

Reply:
About the latest one: https://docs.nethserver.org/en/v7/accounts.html#bind-to-a-remote-ldap-server

That section must be adjusted for the new Cockpit UI. We could add the note about StartTLS there and a link the section here.

Reply:
I agree